labs/vstsextend/Jenkins/README.md

```diff
@@ -114,192 +87,204 @@ Read the complete specifications at [https://docs.microsoft.com/en-us/azure/arch
 
-    ![Jenkins Settings in Team Build](images/vsts-buildjenkinssettings.png)
+1. select **Hosted VS2017** for the Agent Queue, provide **MyShuttle** as the Job name and then select the Jenkins service endpoint created earlier.
 
-1. Next, select the **Get Sources** step. Since Jenkins is being used for the build, there is no need to download the source code to the VSTS build agent. Enable the **Advanced settings** option and select the **Don't sync sources** option.
+    <img class="myImg" src="images/vsts-buildjenkinssettings.png" alt="Jenkins Settings in Team Build" />
 
-    ![Get Sources Settings in Team Build](images/vsts-getsourcessettings.png)
+1. Next, select the **Get Sources** step. Since Jenkins is being used for the build, there is no need to download the source code to the build agent. To skip syncing with the agent, select **Don't sync sources** option.
+
+    <img class="myImg" src="images/vsts-getsourcessettings.png" alt="Get Sources Settings in Team Build" />
 
 1. Next, select the **Queue Jenkins Job** step. This task queues the job on the Jenkins server. Make sure that the services endpoint and the job name are correct. The **Capture console output** and the **Capture pipeline output** options available at this step will be selected.
 
-     ![Jenkins Settings in Team Build](images/vsts-buildjenkinssettings1.png)
+     <img class="myImg" src="images/vsts-buildjenkinssettings1.png" alt="Jenkins Settings in Team Build" />
 
-     >The **Capture console output and wait for completion** option, when selected, will capture the output of the Jenkins build console when the VSTS build runs. The build will wait until the Jenkins Job is completed. The **Capture pipeline output and wait for pipeline completion** option is very similar but applies to Jenkins pipelines (a build that has more than one job nested together).
+     >The **Capture console output and wait for completion** option, when selected, will capture the output of the Jenkins build console when the Azure build pipeline runs. The build will wait until the Jenkins Job is completed. The **Capture pipeline output and wait for pipeline completion** option is very similar but applies to Jenkins pipelines (a build that has more than one job nested together).
 
 1. The **Jenkins Download Artifacts** task will download the build artifacts from the Jenkins job to the staging directory
 
-    ![Download Jenkins Artifact](images/downloadjenkinsartifact.png)
+    <img class="myImg" src="images/downloadjenkinsartifact.png" alt="Download Jenkins Artifact" />
 
-1. The **Publish Artifact drop** will publish to the VSTS.
+1. The **Publish Artifact drop** will publish to Azure Pipelines.
 
-1. Click on the **Save & queue** button to complete the build definition configuration and initiate a new build.
+1. Click **Save & queue** button to save and initiate a new build.
 
 ## Deploying Jenkins Artifacts with Release Management
```

Comment:
Since this exercise is not about deploying to Azure, the heading should change to something like - "Adding Jenkins artifacts in Release Pipeline".